examples/chart/teleport-cluster/charts/teleport-operator/values.yaml

```diff
@@ -0,0 +1,71 @@
+enabled: true
+
+authServer: ""
+caPins: []
+joinMethod: ""
```

Comment:
I'd like to set the default join method to kubernetes as it will be the right choice (in-cluster or JWKS) 99% of the time

Reply:
I think we should do that. tbot and tokens aren't very good friends so it's actually better to orient users to use kubernetes with in-cluster or jwks tokens.

We can add that as a comment